content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -173,11 +166,6 @@ featureGates:
   <feature>: <bool>
 ```
 
-### Adding kube-proxy parameters {#kube-proxy}
-
-For information about kube-proxy parameters in the MasterConfiguration see:
-- [kube-proxy](https://godoc.org/k8s.io/kubernetes/pkg/proxy/apis/kubeproxyconfig/v1alpha1#KubeProxyConfiguration)
-
```

Comment:
see [1]

Reply:
If you want to make it a footnote you also have to put the footnote in... if I am understanding this correctly.